README.md

```diff
@@ -26,19 +26,7 @@ shfmt -i 2 -ci -bn -l -w .
 
 ### Contributing
 
-Check out the **[contributing wiki](https://github.com/pion/webrtc/wiki/Contributing)** to join the group of amazing people making this project possible:
-
-- [Sean DuBois](https://github.com/Sean-Der) - _Original Author of the lint scripts_
-- [Max Hawkins](https://github.com/maxhawkins) - _Git hooks_
```

Comment:
Missing in new contributor list

Reply:
I updated the README again to keep Max in the list and also add a link to the automatically generated `AUTHORS.txt`